change/@internal-react-components-dfd490b8-84bc-430e-bf0f-e32b4877904a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Update internal FileCard component to not show progress bar when progress is 0",
```

Comment:
nitty nitty nit: Shorter change comment means less editing later: `FileCard: Do not show progress bar when progress is 0`.